src/app/components/admin-form-create-event/admin-form-create-event.component.html

```diff
@@ -0,0 +1,72 @@
+
+<mat-tab-group>
+    <mat-tab label="Create Event" (click)="createEvent"> </mat-tab>
+    <mat-tab label="Manage Event" (click)="goToManage"> </mat-tab>
+  </mat-tab-group>
+
+  <p>Welcome, User!</p>
+ 
+
+  <mat-horizontal-stepper linear #stepper>
+    <mat-step>
+        <form [formGroup]="nameFormGroup">
+            <ng-template matStepLabel>Event Name</ng-template>
+            Event Name:
+            <mat-form-field>
+                <mat-label>
```

Comment:
mat-label shouldn't surround the input, they house the label for the input which with angular material is displayed inside the input almost like a place holder.
so,
<mat-label>
Event Name
</mat-label>
<input>

Reply:
gotcha, thanks!